rules/linux/privilege_escalation_container_util_misconfiguration.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2023/07/31"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/31"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the execution of processes that interact with Linux containers through an interactive shell 
+without root permissions. Utilities such as runc and ctr are universal command-line utilities leveraged to interact 
+with containers via root permissions. On systems where the access to these utilities are misconfigured, attackers might 
+be able to create and run a container that mounts the root folder or spawn a privileged container vulnerable to a 
+container escape attack, which might allow them to escalate privileges and gain further access onto the host file system.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Privilege Escalation via Container Misconfiguration"
+references = [
+    "https://book.hacktricks.xyz/linux-hardening/privilege-escalation/runc-privilege-escalation",
+    "https://book.hacktricks.xyz/linux-hardening/privilege-escalation/containerd-ctr-privilege-escalation"
+    ]
+risk_score = 47
+rule_id = "afe6b0eb-dd9d-4922-b08a-1910124d524d"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Privilege Escalation", "Domain: Container"]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+process where host.os.type == "linux" and event.action == "exec" and event.type == "start" and (
+  (process.name == "runc" and process.args : "run") or
+  (process.name == "ctr" and process.args : "run" and process.args : ("--privileged", "--mount"))
+) and not user.Ext.real.id : "0" and not group.Ext.real.id : "0" and 
+process.interactive == true and process.parent.interactive == true
```

Comment:
are these fields dependent on session data? If so, we may want to add it in `setup`

Reply:
Good call, totally forgot about that. Added a small setup guide to the rule!